x-pack/plugins/ingest_manager/server/services/epm/packages/get.ts

```diff
@@ -6,12 +6,7 @@
 
 import { SavedObjectsClientContract } from 'src/core/server/';
 import { PACKAGES_SAVED_OBJECT_TYPE } from '../../../../common/constants';
-import {
-  Installation,
-  Installed,
-  NotInstalled,
-  InstallationStatus,
-} from '../../../../common/types';
+import { Installation, InstallationStatus, PackageInfo } from '../../../../common/types';
```

Comment:
Can we import these constants & types into `server/constants/` and `server/types;`, and import them from there? In general I'd like to avoid deep traversals into `common/` from individual files in server & public.